docs/reference/aggregations/metrics/geocentroid-aggregation.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[search-aggregations-metrics-geocentroid-aggregation]]
 === Geo Centroid Aggregation
 
-A metric aggregation that computes the weighted https://en.wikipedia.org/wiki/Centroid[centroid] from all coordinate values for a <<geo-point>> field.
+A metric aggregation that computes the weighted https://en.wikipedia.org/wiki/Centroid[centroid] from all coordinate values for geo fields.
```

Comment:
Should it be more explicit about geo fields?

```
... coordinate values for a <<geo-point>> or  <<geo-shape>> field.
```

Reply:
I explicitly changed this because I did not want the OSS docs to make it sound like only geo-point is supported. At the same time, I do not want to mix OSS with non-OSS documentation. So saying geo-shape felt misleading. If you disagree, then I'm happy to make things more explicit as you mention

Reply:
good point, let keep it as it is, thanks!